### PR TITLE
[DOCFIX] Improve HDFS and NFS doc

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Configuring-Alluxio-with-NFS.md [hunk 1]
```diff
@@ -14,15 +14,22 @@ First, the Alluxio binaries must be on your machine. You can either
 [compile Alluxio](Building-Alluxio-Master-Branch.html), or
 [download the binaries locally](Running-Alluxio-Locally.html).
 
-Then, if you haven't already done so, create your configuration file from the template:
+Then, if you haven't already done so, create your configuration file with `bootstrapConf` command.
+For example, if you are running Alluxio on your local machine, `ALLUXIO_MASTER_HOSTNAME` should be set to `localhost`
```
Comment (gpang) [inline docs/en/Configuring-Alluxio-with-NFS.md]: Do you mean you want users to replace  `<ALLUXIO_MASTER_HOSTNAME>` with `localhost` (in the following command)? Or do you want users to actually set `ALLUXIO_MASTER_HOSTNAME=localhost`?

Comment (apc999) [inline docs/en/Configuring-Alluxio-with-NFS.md]: I mean the value in the command. If you think this is not clear, we should update all occurrences of this pattern as it is used multiple places across our docs

Comment (gpang) [inline docs/en/Configuring-Alluxio-with-NFS.md]: I think it is slightly confusing, but for now, just update this document.

**Comment by apc999** [comment]
@gpang PTAL.

File: docs/en/Configuring-Alluxio-with-NFS.md (hunk 1 above)
Comment (apc999) [inline docs/en/Configuring-Alluxio-with-NFS.md]: updated

**Pull request merged**